libraries/standard/mqtt/utest/mqtt_utest.c

```diff
@@ -814,6 +849,29 @@ void test_MQTT_Connect_happy_path()
     status = MQTT_Connect( &mqttContext, &connectInfo, &willInfo, timeout, &sessionPresent );
     TEST_ASSERT_EQUAL_INT( MQTTSuccess, status );
     TEST_ASSERT_EQUAL_INT( MQTTConnected, mqttContext.connectStatus );
+
+    /* CONNACK receive with retries. */
```

Comment:
This seems to be the test case where the first call to transport receive is successful. If so, should the comment say "CONNACK receive without retry execution."?

Reply:
What I meant is CONNACK receive using the retry logic. I think it is confusing. Will update the comments. Thanks